The one that updates article meta information

sturobson opened the pull request:
This PR updates the meta information for articles component. 

- removes any usage of `vf-text--` classnames and applies component specific typography as needed
- moves around the content order a little
- makes use of the `vf-links-list` `--tight` variant

todo (when `vf-summary` gets merged and updated) - replace the avatar with `vf-summary--profile` smaller variant.

sturobson committed:
updates to vf-article-meta-informaiton

We have updated the article meta information panel to make use of existing lists, removal of element specific classnames, moving of some content so it's in a better order

sturobson committed:
updates to vf-article-meta-informaiton

We have updated the article meta information panel to make use of existing lists, removal of element specific classnames, moving of some content so it's in a better order

khawkins98 commented on components/vf-article-meta-information/vf-article-meta-information.scss:
```diff
@@ -107,9 +106,8 @@
 .vf-meta__comments {
-
-  .vf-text-body--6 {
+  p {
```
does this have to be a vanilla `p` tag, can we do a class or change the html structure?

sturobson replied:
resolved that in last commit - b6fc7cf

khawkins98 replied:
👍

khawkins98 commented:
We'll probably get bitten by `:visited` links looking a bit silly here, but I suppose that's _really_ an issue with needing to only have purple visited links in long-form content areas. I think that's what #726 is about

sturobson committed:
give element a class

there was a languishing p tag without a class. We have added a class to it and also applied CSS to it directly

sturobson commented:
re: link colours - yes - more than likely there'll be some oddities until this gets resolved
